<?xml version="1.0"?>
<launch>

  <group ns="rokae_arm">
    <rosparam command="load" file="$(find rokae_jps_navigation)/config/rokae_config_JPS.yaml"/>
  </group>

  <node name="rokae_joint2pose" pkg="rokae_jps_navigation" type="rokae_joint2pose" output="screen"/>

  <node name="eef_state" pkg="rokae_jps_navigation" type="rokae_arm_eef_state" output="screen"/>

  <node name="collision_checker" pkg="rokae_jps_navigation" type="rokae_collision_detection" output="screen"/>
  
  <node name="JPS_Planner" pkg="rokae_jps_navigation" type="rokae_jps_planner" output="screen"/>
  <![CDATA[ 
  <node name="JPS_Planner" pkg="rokae_jps_navigation" type="rokae_jps_planner" output="screen" launch-prefix="xterm -e gdb -ex run --args "/>
  ]]>
  <include file="$(find rokae_arm_toppra)/launch/toppra_trajectory.launch"/>
  
</launch>